src/Behat/Config/Formatter/Formatter.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Behat\Config\Formatter;
+
+use Behat\Config\Profile;
+use Behat\Testwork\Output\Printer\Factory\OutputFactory;
+
+class Formatter implements FormatterConfigInterface
+{
+    public function __construct(
+        private readonly string $name,
+        private array $settings = [],
+    ) {
+    }
+
+    /**
+     * @param int $level use Behat\Testwork\Output\Printer\Factory\OutputFactory::VERBOSITY_*
```

Comment:
I think that since you are already importing this name, you can use its short form `OutputFactory::VERBOSITY_*` here

Reply:
done